features/networking/sdn.feature

```diff
@@ -358,3 +358,36 @@ Feature: SDN related networking scenarios
     | nmcli \| grep veth \| wc -l |
   And evaluation of `@result[:response].split("\n")[0]` is stored in the :no_of_veths clipboard
   Then the expression should be true> cb.no_of_veths==cb.no_of_unmanaged_veths
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-29299
+  @admin
```

Comment:
LGTM except this seems to be a @destructive candidate due to CNO patching involved irrespective of whether the behavior passes or fail. Thanks @huiran0826

Reply:
Thanks @anuragthehatter , updated it.